app/services/umrdr/deposits_okay_service.rb

```diff
@@ -0,0 +1,59 @@
+module Umrdr
+
+  require 'rubygems'
+  require 'net/ldap'
+
+  ENV['LDAPTLS_CACERT'] = '../../../config/incommon_server_ca_cert.crt';
+
+  class DepositsOkayService
```

Comment:
I think we'll be using this service to determine if deposit is permitted, but that's a step ahead.  A more precise name would probably be more apropos for this class.  Can the precise function of this class be distilled? It takes a uniquename and queries our ldap for the user's roles.